packages/server/modules/shared/helpers/dbHelper.ts

```diff
@@ -106,6 +106,9 @@ export const numberOfFreeConnections = (knex: Knex) => {
   return Math.max(pgMaxConnections - demand, 0)
 }
 
+/**
+ * @deprecated use withOperationTransaction for a better API
```

Comment:
requires you to manually create the trx beforehand which makes it less useful than it could be. the trx object should be an internal implementation detail, you don't need it outside of the function